src/olympia/blocklist/tasks.py

```diff
@@ -60,25 +65,33 @@ def import_block_from_blocklist(record):
         guid_regexp = bracket_open_regex.sub(r'\{', guid[1:-1])
         guid_regexp = bracket_close_regex.sub(r'\}', guid_regexp)
         log.debug(
-            'Attempting to create Blocks for addons matching [%s]',
-            guid_regexp)
+            'Kinto %s: Attempting to create Blocks for addons matching [%s]',
+            kinto_id, guid_regexp)
         addons_qs = Addon.unfiltered.filter(guid__regex=guid_regexp)
         # We need to mark this id in a way so we know its from a
         # regex guid - otherwise we might accidentally overwrite it.
         block_kw['kinto_id'] = '*' + block_kw['kinto_id']
+        regex = True
     else:
-        log.debug('Attempting to create a Block for guid [%s]', guid)
+        log.debug(
+            'Kinto %s: Attempting to create a Block for guid [%s]',
+            kinto_id, guid)
         addons_qs = Addon.unfiltered.filter(guid=guid)
+        regex = False
     for addon in addons_qs:
         (block, created) = Block.objects.update_or_create(
             guid=addon.guid,
             defaults=dict(guid=addon.guid, **block_kw))
         block_activity_log_save(block, change=not created)
         if created:
-            log.debug('Added Block for [%s]', block.guid)
+            log.debug('Kinto %s: Added Block for [%s]', kinto_id, block.guid)
             block.update(modified=modified_date)
         else:
-            log.debug('Updated Block for [%s]', block.guid)
-    else:
+            log.debug('Kinto %s: Updated Block for [%s]', kinto_id, block.guid)
+    if addons_qs:
+        kin.update(outcome=(KintoImport.OUTCOME_REGEXBLOCKS if regex else
+                            KintoImport.OUTCOME_BLOCK))
+    if not addons_qs:
```

Comment:
Could this just be an `else` paired with the `if addons_qs:` above?

Reply:
it should!

Reply:
https://github.com/mozilla/addons-server/pull/13274/commits/05b0c2536b4bf1b1f310a482ad84345acf39477d